internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -2,8 +2,9 @@
 # SPDX-License-Identifier: MIT-0
 AWSTemplateFormatVersion: 2010-09-09
 Description: CloudFormation template that represents a load balanced web service on Amazon ECS.
-{{- if .SerializedManifest }}
 Metadata:
+  Version: {{ .Version }}
```

Comment:
i'm trying to think about what happens in the case when someone upgrades the CLI, runs `svc deploy`, and the only thing that changes about the template is this field. I assume that will return an empty change set error and not actually update the template?

assume this ⬆️ is true, then you can do something like this, assuming no changes template changes other than `Metadata.Version` between v1.40 and v1.42:
```
copilot@v1.40 svc deploy
copilot@v1.42 svc deploy # empty change set, template not updated
copilot@v1.41 svc deploy # no version error - just empty change set again, since the template didn't actually upgraded to v1.42
copilot@v1.39 svc deploy # not allowed due to template downgrade
```

i think this is actually pretty great - i'm just validating that's how this will work?

Reply:
Yeah i think wx tried to do the same thing for env template. I think should be fine?